restord/ImageSessionServer.m

```diff
@@ -358,6 +359,11 @@ - (void)processOutput:(NSData *)output {
                ![line isEqualToString:@"done"] &&
                ![line isEqualToString:@"Validating target..."] &&
                ![line isEqualToString:@"Validating sizes..."]) {
+      // Print personalization successes
+      if ([line isEqualToString:@"Personalization over TDM succeeded"]) {
```

Comment:
why is this within another if statement? it would make more sense as an elseif clause on line 356?